playbooks/provisioner/virsh/cleanup.yml

```diff
@@ -62,10 +62,11 @@
         with_items: "{{ provisioner.topology.network.keys() }}"
 
       - name: remove any existing disks that we created
-        shell: "rm -f {{ item.key }}*disk*.qcow*"
-        args:
-            chdir: "/var/lib/libvirt/images"
-        with_dict: "{{ provisioner.topology.nodes }}"
```

Comment:
Someone who wrote virsh provisioner should check if it's properly cleaning the resources as the logic here is slightly modified.